popup/popup.js

```diff
@@ -1,42 +1,48 @@
 var toastActive = false;
 function triggerToast () {
-  if (!toastActive) {
-    toastActive = true;
-    Materialize.toast('Refresh to see changes!', 4000, '', function() {
-      toastActive = false
-    });
-  }
+  if (toastActive)
+    return;
+
+  toastActive = true;
+  Materialize.toast('Refresh to see changes!', 4000, '', function() {
+    toastActive = false
+  });
 }
 
-$(function() {
-  $("button.color").click(function() {
-    var color = $(this).css('background-color');
+Array.prototype.forEach.call(document.querySelectorAll("button.color"), function(button) {
```

Comment:
Thanks for replacing the other instances with `Array.from`. Could you also do it for this line?